src/sdk/storage/identityVerificationStorageRepository.ts

```diff
@@ -81,4 +81,12 @@ export class IdentityVerificationStorageRepository {
             },
         });
     }
+
+    public async findCountByType(type: VerificationTypeEnum): Promise<number> {
+        return await this.ormRepository.count({ where: { type } });
```

Comment:
this is not correct. you need to do a sum of the `reuseCount` value of all found records.  currently the `count()` you have implement will just count the number of records, not the sum of the values.

similar on L89